src/app/pages/table-page/basic-table.component.ts

```diff
@@ -1,38 +1,46 @@
 import {
   CUSTOM_ELEMENTS_SCHEMA,
   Component,
-  ElementRef,
-  ViewChild,
-  inject,
-  Renderer2,
-  AfterViewInit, Input
-} from '@angular/core'
-import {TableData} from './table-data'
+} from '@angular/core';
+import exampleData from './exampleData.json';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-basic-table',
   template: `
     <h1>Basic Table</h1>
-    <tds-table vertical-dividers="false" compact-design="false" enable-responsive="false">
+    <tds-table vertical-dividers="false" compact-design="false" responsive="false">
       <tds-table-header>
-        <tds-header-cell column-key="truck" column-title="Truck type"></tds-header-cell>
-        <tds-header-cell column-key="driver" column-title="Driver name"></tds-header-cell>
-        <tds-header-cell column-key="country" column-title="Country"></tds-header-cell>
-        <tds-header-cell column-key="mileage" column-title="Mileage"></tds-header-cell>
+          <tds-header-cell column-key="truck" column-title="Truck type"></tds-header-cell>
+          <tds-header-cell column-key="driver" column-title="Driver name"></tds-header-cell>
+          <tds-header-cell column-key="country" column-title="Country"></tds-header-cell>
+          <tds-header-cell column-key="mileage" column-title="Mileage"></tds-header-cell>
       </tds-table-header>
-      <tds-table-body #basicTableBody></tds-table-body>
+      <tds-table-body>
+          <tds-table-body-row *ngFor="let row of tableData;">
+              <tds-body-cell cell-key="truck">
+                  {{ row.truck }}
```

Comment:
Will this break since we're using a default slot?

Reply:
I'm not sure how normal it would be. 
Might be a trade-off we can make.
Forcing users to put another div in there would pain me - since theres already a lot of nesting

Reply:
We can introduce a named slot post 1.0. Will leave as is for now.